runtime/lua/vim/inspect.lua

```diff
@@ -1,7 +1,7 @@
-local inspect ={
-  _VERSION = 'inspect.lua 3.1.0',
-  _URL     = 'http://github.com/kikito/inspect.lua',
-  _DESCRIPTION = 'human-readable representations of tables',
+local inspect = {
+  _VERSION = "inspect.lua 3.1.0",
```

Comment:
should the version be updated?

Reply:
By upstream, yes. (https://github.com/kikito/inspect.lua/blob/9c8a68da3033f1393b84b498313572654af82b8f/inspect.lua#L20)